docs/bom/README.md

```diff
@@ -0,0 +1,51 @@
+# bom (Bill of Materials)
+
+Create SPDX compliant Bill of Materials 
+
+- [Summary](#summary)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Summary 
+
+bom is a little utility that lets software authors generate
+SPDX manifests to describe the contents of a release. The
+SPDX manifests provide a way to list and verify all items
+contained in packages, images, and individual files while
+packing the data along with licensing information.
+
+bom is still in its early stages and it is an effort to open
+the libraries developed for the Kubernetes SBOM for other 
```

Comment:
I made suggestions on BOM / SBOM references but generally I have mixed feelings on the usage of them. On one hand it increases search hits to use both, but on the other I think we should be consistent on which term to use. I'm leaning on saying SBOM for sake of consistency, since we already mention SBOM in the document for "What is Bill of Materials".

Reply:
+1 I've noticed SBOM gaining more acceptance in discussions happening everywhere. Following up on @saschagrunert comment someplace else, I think we should settle on:
`bom` for the binary
SBOM to refer to a software bill of materials in general.